src/main/java/de/tum/cit/ase/ares/api/archunit/SecurityRules.java

```diff
@@ -0,0 +1,56 @@
+package de.tum.cit.ase.ares.api.archunit;
+
+import com.tngtech.archunit.base.DescribedPredicate;
+import com.tngtech.archunit.core.domain.*;
+import com.tngtech.archunit.lang.ArchRule;
+import com.tngtech.archunit.lang.syntax.ArchRuleDefinition;
+import de.tum.cit.ase.ares.api.archunit.conditions.TransitivelyAccessesMethodsCondition;
+import org.apiguardian.api.API;
+
+import java.util.*;
+
+/**
+ * This class executes the security rules on the architecture.
+ */
+@API(status = API.Status.INTERNAL)
+public class SecurityRules {
+
+    private SecurityRules() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    /**
+     * This method checks if any class in the given package accesses the file system.
+     *
+     * @return The ArchRule to check for file system access.
+     */
+    public static ArchRule noClassesShouldAccessFileSystem(String studentPackage) {
+        // Packages that should not be accessed
+        List<String> bannedPackages = List.of(
+                "java.nio.file",
+                "java.util.prefs",
+                "sun.print",
+                "sun.security",
+                "java.util.jar",
+                "sun.awt.X11",
+                "javax.imageio.stream",
+                "javax.sound.midi",
+                "javax.swing.filechooser",
+                "java.awt.desktop");
+
+        return ArchRuleDefinition.noClasses()
+                .that()
+                .resideInAPackage(studentPackage + "..")
```

Comment:
same issue here, none of the analyzed classes are allowed to access the file system.
```suggestion
```